### PR TITLE
feat: Allow everyone to refresh an entity

### LINKED ISSUE #54: Bug: Refresh button on Component details page results in permission error
Clicking the refresh button results in following error:

- [x] fix
- [x] deploy in a new release

image

### PR DESCRIPTION
**Pull request by SamoKopecky**
Resolves #54 

I also reworked the permission policy a bit to make future permission changes easier.

### CODE REVIEW

File: packages/backend/src/plugins/permission.ts [hunk 1]
```diff
@@ -13,8 +13,12 @@ import { PluginEnvironment } from '../types';
 
 class TestPermissionPolicy implements PermissionPolicy {
   async handle(request: PolicyQuery): Promise<PolicyDecision> {
-    if (request.permission.name === 'catalog.entity.read') {
-      return { result: AuthorizeResult.ALLOW }
+    const permissions: Array<string> = [
```
Comment (tumido) [inline packages/backend/src/plugins/permission.ts:16]: The type should be automatically inferred here, isn't it? I don't think you need to specify it.

**Review by tumido — COMMENTED** [review]
A tiny nit, if you say you don't feel like changing it, I'll let it merge as is.

File: packages/backend/src/plugins/permission.ts (hunk 1 above)
Comment (SamoKopecky) [reply packages/backend/src/plugins/permission.ts:16]: I can change this, it looks cleaner without the type anyways.

Comment (SamoKopecky) [reply packages/backend/src/plugins/permission.ts]: Fixed in the latest commit.

**Comment by SamoKopecky** [comment]
I just thought of an idea, wouldn't it be better if we make our own config file for permissions and just read permissions from the config file instead of defining it in code?

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**